spec/std/socket/spec_helper.cr

```diff
@@ -13,6 +13,6 @@ def each_ip_family(&block : Socket::Family, String, String ->)
   end
 
   describe "using IPv6" do
-    block.call Socket::Family::INET6, "::1", "::"
+    block.call Socket::Family::INET6, "[::1]", "::"
```

Comment:
Why change this spec? It's at most unrelated.

Reply:
By the way, should we chop the brackets or should every socket type constructor should support the IPv6 bracket syntax out of the box (not only just WebSocket)?